frontend/sidedish/src/BigSidedish.js

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import "./BigSidedish.css";
+
+function SidedischCard(props) {
```

Comment:
```suggestion
function SidedischCard() {
```

사용하지 않는 인자라면 우선 지워주시고, 나중에 추가해주시는 게 가독성에 있어 좋을 거 같아요(git이 있으니까요).